### PR TITLE
Run only one test per commit

### PR DESCRIPTION
**Pull request by psharda**
#317 used 2 cores to run azure pipelines, but they trigger 2 tests each, so each commit is being tested four times. This PR can ensure each commit only triggers one test but uses both cores.

### CODE REVIEW

**Comment by BenWibking** [comment]
The arm64 debug CI build failed.

File: .ci/azure-pipelines-aarch64-debug.yml [hunk 1]
```diff
@@ -22,15 +22,11 @@ jobs:
     inputs:
       cmakeArgs: '.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DAMReX_SPACEDIM=1'
 
-  - task: CMake@1
+  - script: cmake --build .
     displayName: 'Build Quokka'
-    inputs:
-      cmakeArgs: '--build .'
```
Comment (BenWibking) [inline .ci/azure-pipelines-aarch64-debug.yml:28]: Is this change necessary?

File: .ci/azure-pipelines-aarch64-debug.yml [hunk 2]
```diff
@@ -22,15 +22,11 @@ jobs:
...
 
-  - task: CMake@1
+  - script: ctest -j 2 -I 1,1 -T Test --output-on-failure
```
Comment (BenWibking) [inline .ci/azure-pipelines-aarch64-debug.yml:28]: Can you change the arguments in `cmakeArgs` below instead?

**Comment by psharda** [comment]
Doesn't look like we need it anymore, since recent pushes to other open PRs only trigger two tests per commit (as before).
